### DISCUSSION TITLE
What are the units of shapes generated in Pyvista?

### DISCUSSION THREAD

**Discussion by harshamarupudi56** [pyvista/pyvista#3149] [Q&A] [answered]
I want to generate a cylinder with pyvista and then turn that into a mesh. 

https://docs.pyvista.org/api/utilities/_autosummary/pyvista.Cylinder.html

Looking at this example I see that the user can define the radius and height of the cylinder. But what are the units here?

**Comment by tkoyama010** [comment, marked as answer]
PyVista objects do not have units. It is easier to understand if you imagine geometry in mathematics.

**Reply by banesullivan** [reply]
Further: Units are implicit - whatever you, as the user, decide are the units will be the units.

**Comment by harshamarupudi56** [comment]
Understood thank you for the response.
